### PR TITLE
Re-format schedules for AKN3

### PR DESCRIPTION
**Pull request by longhotsummer**
* move under `act`
* component[s] -> attachment[s]
* move headings
* remove hcontainer

### CODE REVIEW

File: indigo_api/tests/data_migrations/test_schedules_to_attachments.py [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+  </components>
+</akomaNtoso>
+    """)
+        migration.migrate_document(doc)
+        xml = doc.doc.to_xml(pretty_print=True, encoding='unicode')
+        self.assertMultiLineEqual("""<akomaNtoso xmlns="http://www.akomantoso.org/2.0">
+  <act contains="originalVersion">
+    <meta/>
+    <body/>
+  <attachments>
```
Comment (goose-life) [inline indigo_api/tests/data_migrations/test_schedules_to_attachments.py:67]: is it okay that this comes back out even though the <act> tag isn't closed yet?

File: indigo_api/tests/data_migrations/test_schedules_to_attachments.py [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+              <content>
+                <p>hi</p>
+              </content>
+            </paragraph>
+          </mainBody>
+      </doc>
+    </attachment>
+  </attachments>
+</act>
+  </akomaNtoso>
```
Comment (goose-life) [inline indigo_api/tests/data_migrations/test_schedules_to_attachments.py:100]: is this indentation also okay?

File: indigo_api/tests/data_migrations/test_schedules_to_attachments.py (hunk 1 above)
Comment (longhotsummer) [reply indigo_api/tests/data_migrations/test_schedules_to_attachments.py:67]: Yes, the formatting is largely academic. In reality, the document won't contain whitespace. This is just to try to make understanding text failures easier.

File: indigo_api/tests/data_migrations/test_schedules_to_attachments.py (hunk 2 above)
Comment (longhotsummer) [reply indigo_api/tests/data_migrations/test_schedules_to_attachments.py:100]: yes. Annoying, but okay.

**Comment by goose-life** [comment]
I've merged this branch into https://github.com/laws-africa/indigo/pull/1079

**Pull request closed without merging**